### ISSUE TITLE
[BO] Bad display of the header search box in new theme for RTL languages

### ISSUE THREAD

**Issue by mparvazi** [PrestaShop/PrestaShop#26635]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

The header search box in new theme pages (e.g. Orders) has oversized height. This will happen when the input is in the blur mode (not focused).

<img src="https://user-images.githubusercontent.com/85633460/142036140-2640fb06-f9e5-47b0-908e-210e48396612.png" width="700"/>

### Expected behavior

<img src="https://user-images.githubusercontent.com/85633460/142037066-ea1f6a75-5823-4ec6-991a-b31fe8b5188a.png" width="700"/>

### Steps to reproduce

1. Go to BO > Orders

### PrestaShop version(s) where the bug happened

1.7.8.1

### PHP version(s) where the bug happened

7.4

**Comment by mparvazi** [comment]
@NeOMakinG 

In blur mode (not focused), the search type button (dropdown menu) is hidden.
This code somehow works fine for LTR languages:
```
.main-header #header-search-container form.collapsed .btn {
    width: 0;
    padding: 0;
    overflow: hidden;
    border: 0;
    opacity: 0;
}
```
But for RTL languages this rule must be added:
```
.main-header #header-search-container form.collapsed .btn {
+   height: 0;
}
```

It's related to RTL font characters, there is empty spaces after the text and causes the problem.
Because the height isn't defined then the overflow rule doesn't work here.

This problem also occurs in LTR, but it is very small (just one pixel overflow).

I think adding `height: 0;` will be enough and works fine for LTR and RTL.
```
admin-dev\new-theme\scss\components\layout\_main_header.scss(231)
```

It’s tempting to use `display: none;` instead of all these rules, but it isn't possible because of the `transition` effect!

**Comment by hibatallahAouadni** [comment]
Hello @mparvazi 

Thanks for reporting this issue!
As discussed on Slack, it's a regression (on 1.7.8.0) only with RTL languages, see the attached screenshot below:

![image](https://user-images.githubusercontent.com/23308427/142172681-3c6b2c2d-5986-46ea-bc01-40754af880f9.png)

https://watch.screencastify.com/v/e09N4qRsvt3oY8d2hnCm

As you can see (in the screen record above) the issue is gone when you click on the search input, but it return when you refresh the page. I’ll add this to the debug backlog so that it’s fixed.

If you manage to get the problem fixed, then please contribute the fix back to the project as a [Pull Request](https://devdocs.prestashop.com/1.7/contribute/contribute-pull-requests/). Remember that the more people contribute, the better PrestaShop becomes for everyone.

Thank you

**Comment by mparvazi** [comment]
Hello @hibatallahAouadni 

I think this issue is not only related to RTL languages, it's just more visible in RTL.

This problem also occurs in LTR, but it is very small (just one pixel overflow).

**Comment by Progi1984** [comment]
Fixed by @mparvazi in #26691
